policies/broken-nightlies.md

```diff
@@ -0,0 +1,30 @@
+# Policy on broken nightlies
+
+Sometimes the nightlies released automatically by our CI ends up being broken
+for some people or even everyone. This policy defines what the infra team
+response will be in those cases.
+
+## Which nightly will be rolled back
+
+* A nightly can be rolled back if it contains destructive code, for
+  example if the included compiler deletes all the users files.
+
+* A nightly can be rolled back if an infra problem caused it to be broken for a
+  big percentage of users on any Tier 1 platform. Issues affecting only lower
+  tier platforms are not worthy of a roll back, since we don't guarantee working
+  builds for those platforms anyway.
+
+* A nightly will **not** rolled back if it contains a compiler bug, since those
+  bugs are expected to be caught by CI.
+
+## What are we going to fix
+
+When the infra team decides a nightly will be rolled back, we will only fix
```

Comment:
Should this go in more detail on what it means to roll back?  I assume it means `rustup toolchain install nightly` will install the previous nightly rather than the broken one, "as if yesterday's build never happened".  What do users that updated nightly before the rollback have to do to rollback?

Reply:
Clarified the wording here. I don't think the implementation details of the roll back (like "What do users that updated nightly before the rollback have to do to rollback?") have to go into a policy though.

Reply:
That's true.